package.json

```diff
@@ -81,7 +81,7 @@
     "flatley": "^5.2.0",
```

Comment:
I don't see any refactor ? unlike what is indicated in the content. You forgot to push some files ?

Reply:
i mean jsonld did some refactoring